lib/components/atoms/Blockquote/Blockquote.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+/**
+ *
+ * Blockquote
+ *
+ */
+import React from 'react';
+import styled from 'styled-components';
+
+import styles from './Blockquote.style';
+import type { Props } from './types';
+
+const Blockquote = ({ className, quoteProp, quoteAuthorProp }: Props): Node => (
+  <blockquote className={className}>
+    <p className="quoteContent">{quoteProp}</p>
+    <footer className="quoteAuthor">{quoteAuthorProp}</footer>
+  </blockquote>
+);
+
+Blockquote.defaultProps = {
+  quoteProp: '',
```

Comment:
There should not be default value for Quote 

The default values should be meaningful.